docs/rst/tutorials/core/qos/initial_peers/initial_peers.rst

```diff
@@ -0,0 +1,198 @@
+Therefore, this tutorial only make sense if the user has more than one ROS 2 Context.
+
+According to the DDS standard, each DomainParticipant must listen for incoming DomainParticipant discovery metatraffic in two different ports, one linked with a multicast address, and another one linked to a unicast address.
+Vulcanexus, together with *Fast DDS*, allows configuring an initial peers list, which contains one or more such IP-port address pairs corresponding to remote DomainParticipants discovery listening resources, so that the local DomainParticipant will not only send its PDP traffic to the default multicast network address, but also to all the IP-port address pairs specified in the initial peers list.
+
+A complete description of the initial peers list and its configuration can be found in Initial peers |InitialPeersFastDdsDocs|.
+
+.. _initial_peers_prerequisites:
+
+Prerequisites
+-------------
+
+First of all, make sure that Vulcanexus Humble is installed.
+The docker installation is required for this tutorial (see :ref:`Docker installation <docker_installation>`).
+
+Also, complete the :ref:`Configuring Fast-DDS QoS via XML profiles <tutorials_xml_profiles_intro>` tutorial to ensure the execution of the XML profile configuration.
+
+Open two terminals, and run the Vulcanexus Humble image in each one with the following command:
+
+.. code-block:: bash
+
+      docker run \
+        -it \
+        --name ros2_context_1 \ #rename ros2_context_2 in the second case
+        ubuntu-vulcanexus:humble-desktop
+
+
+.. _initial_peers_add_qos:
+
+Adding initial peers QoS Policy
+-------------------------------
+
+In the docker container named ``ros2_context_1``, create a XML file, and complete it with the following example:
+
+.. literalinclude:: /resources/tutorials/core/qos/initial_peers/initial_peers_config.xml
+    :language: xml
+
+.. note::
+
+    For advanced users, and according to the `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_ (Section 9.6.1.1), the participant discovery traffic unicast listening ports are defined in the `Well Known Ports <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/listening_locators.html?highlight=well%20known#well-known-ports>`_ section of the Fast DDS documentation.
+    Thus, in this example, the Participant operates in Domain 0 (default domain) and its ID is 1, so its discovery traffic unicast listening port is 7412.
+    By default *eProsima Fast DDS* uses as initial peers the metatraffic multicast locators (network addresses).
+
+    In the last part of the example it is described how to avoid multicast, using unicast and initial peers simultaneously.
+
+It is required to include the XML file in at least one of the Vulcanexus docker containers.
+As long as they are using multicast to discover each other, the participant with the initial peer set would let the other participant know the address and port through which they will communicate.
```

Comment:
Multicast is just for exchanging the unicast addresses so it should work also without multicast communication. 

```suggestion
It is required to include the XML file in at least one of the Vulcanexus docker containers, as the participant with the initial peer set would let the other participant know the address and port through which they will communicate.
```